packages/components/pie-modal/src/defs.ts

```diff
@@ -1,8 +1,19 @@
 export const headingLevels = ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'] as const;
 
 export interface ModalProps {
+    /**
+     * The text to display in the modal's heading.
+     */
     heading: string;
+    /**
```

Comment:
super nit but you're using full stops in modal but not in button or icon button 😄

Reply:
@xander-marjoram would be proud

Reply:
Ugh. Removing them from here now